lgc/builder/DescBuilder.cpp

```diff
@@ -80,7 +80,7 @@ Value *DescBuilder::CreateLoadBufferDesc(unsigned descSet, unsigned binding, Val
       return UndefValue::get(getBufferDescTy(pointeeTy));
     }
 
-    if (node == topNode && isa<Constant>(descIndex)) {
+    if (/*false &&*/ node == topNode && isa<Constant>(descIndex)) {
```

Comment:
This wants to be "node && node == topNode" doesn't it?

Reply:
This is just for my debugging while the patch is still WiP. I think this check should go into a separate patch, if that makes sense.